### ISSUE TITLE
Classification by a Neural Network using Keras: Pairplot 

### ISSUE THREAD

**Issue by LeaMyCode** [carpentries-incubator/deep-learning-intro#248]
In the material, one has to answer the following question: "Which combination of attributes shows the best separation for all 3 class labels at once?" and the answer shows "combination of bill length and bill depth", but the combination of bill length and flipper length shows the same seperation. 

Link to the material: https://carpentries-incubator.github.io/deep-learning-intro/02-keras/index.html

**Comment by dsmits** [comment]
Yes I think you're right. I suggest we make it clear that there are also other correct answers.
